src/constants/links.js

```diff
@@ -0,0 +1,11 @@
+export const header = {
+  navDropdowns: [
+    ["link-one", "link-two"],
```

Comment:
Should these paths have `/`?

Reply:
Good catch! But it should be how it is right now, otherwise, it would redirect to `/link-one` instead of `/en/link-one`. The footer links don't have any `/` now.